charts/nexus-iq/values.yaml

```diff
@@ -117,10 +121,13 @@ persistence:
   # existingClaim:
   # annotations:
   #  "helm.sh/resource-policy": keep
-  ## Configure the volume type.
+  # storageClass: "-"
   # volumeConfiguration:
   #   hostPath:
   #     path: /data/nxiq/
+  # If PersistentDisk already exists you can create a PV for it by including the 2 following keypairs.
+  # pdName: nexus-data-disk
```

Comment:
i see this whole thing does look a lot like the nxrm chart now 👍 , but I think I'd maybe rename the sample value here to nexus-iq-data-disk or something, so we're not accidentally leading people to overlap disks between the 2 servers when they install both this way.

Reply:
> i see this whole thing does look a lot like the nxrm chart now 👍 , but I think I'd maybe rename the sample value here to nexus-iq-data-disk or something, so we're not accidentally leading people to overlap disks between the 2 servers when they install both this way.

It just serves as an example, and I'm not sure that utilizing the same disk for both products is actually not a serviceable pattern ;)

Reply:
But you forget: this is kubernetes, why share a resource when you could have 3 copies? 😆